Dockerfile

```diff
@@ -0,0 +1,57 @@
+FROM fpco/stack-build:lts-8.18
+
+ENV LANG C.UTF-8
+
+# Install required packages.
+RUN apt-get update
+RUN apt-get upgrade -y --assume-yes
+# Install packages for stack and ghc.
+RUN apt-get install -y --assume-yes xz-utils gcc libgmp-dev zlib1g-dev
+# Install packages needed for libraries used by our app.
+RUN apt-get install -y --assume-yes libmysqlclient-dev
+# Install convenience utilities, like tree, ping, and vim.
+RUN apt-get install -y --assume-yes tree iputils-ping vim-nox
+
+# Remove apt caches to reduce the size of our container.
+RUN rm -rf /var/lib/apt/lists/*
```

Comment:
I kind of like having them separated into their reason for being added, but there are some I should remove looking at the list now.